### ISSUE TITLE
Intermittent test failures

### ISSUE THREAD

**Issue by doctor-g** [spring2017gamestudio/prairie-creek#50]
Sometimes the automatic tests fail with the message "https://travis-ci.org/spring2017gamestudio/prairie-creek/builds/209783753". I saw this today when running the tests once, then made no changes, ran them again, and it didn't come up.

An example of this failure can be seen [in this build](https://travis-ci.org/spring2017gamestudio/prairie-creek/builds/209783753).

**Comment by doctor-g** [comment]
Turns out we still have intermittent failures, though with a slightly different error message:
```
Tests failed: Error thrown outside of test function: document.getElementById(...).restore is not a function
```
You can see it in [this build](https://travis-ci.org/spring2017gamestudio/prairie-creek/builds/211362150), for example.
Previously it was giving this error with the `create` method, not `restore`, I think. I wonder if the fact that it's failing after 14 tests is significant.

**Comment by doctor-g** [comment]
In #52 I have hit this error again, after thinking I fixed it. I made a few changes and haven't seen the error, but I'm not convinced that it's gone: I think it's a nondeterministic timing issue that's giving us the occasional false-negative. I don't think it's quite right to keep this particular issue open, though; we may just have to keep our eyes open for this as we try to triangulate toward a solution.
